### PR TITLE
[documentation] Add Remark how to clean old cobra toolboxes

### PR DESCRIPTION
**Pull request by tpfau**
Add a remark on how to properly clean old cobra toolboxes from the matlab path and the system.

*Please include a short description of enhancement here*

**I hereby confirm that I have:**

- [] Tested my code on my own machine
- [] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -44,10 +44,23 @@ Installation
 ------------
 
 .. begin-installation-marker
+0. If you have an existing installation of an old COBRA toolbox on your system, please remove
```
Comment (laurentheirendt) [inline README.rst]: May you start this counter with `1.`?

File: README.rst [hunk 2]
```diff
@@ -44,10 +44,23 @@ Installation
 ------------
 
 .. begin-installation-marker
+0. If you have an existing installation of an old COBRA toolbox on your system, please remove 
+   the installation and the path  from your folder. You can skip this step, if you do not have
+   an old cobra toolbox installed.
```
Comment (laurentheirendt) [inline README.rst]: `a legacy COBRA Toolbox`

File: README.rst (hunk 1 above)
Comment (laurentheirendt) [inline README.rst]: `a legacy COBRA Toolbox`

File: README.rst [hunk 3]
```diff
@@ -44,10 +44,23 @@ Installation
...
+
+   |warning| The following commands will delete your CobraToolboxFolder and all of its contents
```
Comment (laurentheirendt) [inline README.rst]: `cobratoolbox` directory

File: README.rst [hunk 4]
```diff
@@ -44,10 +44,23 @@ Installation
...
 .. begin-installation-marker
+0. If you have an existing installation of an old COBRA toolbox on your system, please remove 
+   the installation and the path  from your folder. You can skip this step, if you do not have
+   an old cobra toolbox installed.
+
+   |warning| The following commands will delete your CobraToolboxFolder and all of its contents 
+
+   .. code-block:: matlab
+   
+         >> cobraFolder = fileparts(which('initCobraToolbox'));
```
Comment (laurentheirendt) [inline README.rst]: May we call it `CBTDIR` to stay consistent?

File: README.rst (hunk 1 above)
Comment (tpfau) [reply README.rst]: It was to some extent intentional to start with 0, as this is only required if there is an old toolbox. 
Starting with sounds as if it is a step always required.

Comment (laurentheirendt) [reply README.rst]: Then this should be a different number, or explicitly stating that this is an optional step.

**Comment by syarra** [comment]
Thanks @tpfau and @laurentheirendt for adding these instructions in the FAQ. 👍

**Pull request merged**